src/Proto.Actor/Messages/IAutoRespond.cs

```diff
@@ -0,0 +1,12 @@
+// -----------------------------------------------------------------------
+// <copyright file="IMessageAutoAck.cs" company="Asynkron AB">
+//      Copyright (C) 2015-2021 Asynkron AB All rights reserved
+// </copyright>
+// -----------------------------------------------------------------------
+namespace Proto
+{
+    public interface IAutoRespond
```

Comment:
New feature, IAutoRespond. when the actor receives this message in the receive pipeline.
it calls the GetAutoResponse and does a context.Respond(msg.GetAutoResponse()); with that.

this way, we can have Acks on messages/message batches, w/o the actor itself having to care about it